README.md

```diff
@@ -127,6 +127,53 @@ spec:
         hostPath:
           path: /etc/localtime
 ```
+
+For Kubernetes >= 1.9 you need to add selector tag as bellow:
```

Comment:
@jsenon Thanks for updating this to catch up with newest deployment spec.
One nit: Will it be wrong to add selector tag for Kubernetes below 1.9? if it is not, i suggest we add selector to [node-problem-detector](https://github.com/jsenon/node-problem-detector/blob/master/node-problem-detector.yaml)  and directly

Reply:
I don’t have tested selector field below 1.9

Reply:
I have tested this against Kubernetes 1.8. Seems fine.